### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by tgautier**

### CODE REVIEW

**Comment by tgautier** [comment]
I forgot about dogma, we need to add it too.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Circle CI](https://circleci.com/gh/fewlinesco/bamboo_smtp.svg?style=svg)](https://circleci.com/gh/fewlinesco/bamboo_smtp)
+[![Build Status](https://travis-ci.org/fewlinesco/bamboo_smtp.svg?branch=feature%2Fadd_travis_ci)](https://travis-ci.org/fewlinesco/bamboo_smtp)
```
Comment (kdisneur) [inline README.md]: I guess the branch should be master here

**Comment by kdisneur** [comment]
I didn't know Hound supported Elixir. 💭

**Comment by tgautier** [comment]
It doesn't, I've removed it.

**Comment by tgautier** [comment]
@kdisneur I think it's done

**Comment by kdisneur** [comment]
@tgautier : looks fine 👍

**Pull request merged**